RELEASE/scripts/autoscend/quests/level_11.ash

```diff
@@ -2181,32 +2197,52 @@ boolean L11_hiddenCityZones()
 		if (canUseMachete && !equipMachete()) {
 			return false;
 		}
+		if(!canUseMachete && !equipTearawayPants())
```

Comment:
I'd suggest changing to 
if(!canUseMachette && auto_haveTearawayPants())
{
force equip pants
}

This way all paths that can't use a machete and have pants, will use them. This would also mean removing your new function

Reply:
I read this commment after I moved things around. Yes, this should be fine because pokefam's combat is taken care of independently from every other combat